main.go

```diff
@@ -13,9 +14,11 @@ import (
 	"github.com/labstack/echo/v4/middleware"
 	"github.com/rs/zerolog"
 	"github.com/spf13/pflag"
+	"google.golang.org/grpc"
 
 	"github.com/awfm9/flow-dps/chain"
 	"github.com/awfm9/flow-dps/feeder"
+	grpcApi "github.com/awfm9/flow-dps/grpc"
```

Comment:
Let's change the package name to avoid duplication here. What else could we use?

Reply:
No idea, I thought about it for a minute and nothing comes to my mind that would not make it confusing. Maybe it would make more sense to alias the other `grpc` package though, so that our own APIs are consistent (`rest`, `grpc`.).

And then, maybe the Google GRPC package could be aliased to `googlegrpc` or something like that.

Reply:
Let's not waste time on it now, we can clean it up as part of some random future PR.